xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -64,6 +67,30 @@ static int InstallAddon(const std::vector<std::string>& params)
   return 0;
 }
 
+/*! \brief Enable an addon.
+ *  \param params The parameters.
+ *  \details params[0] = add-on id.
+ */
+static int EnableAddon(const std::vector<std::string>& params)
+{
+  const std::string& addonid = params[0];
+
+  if (!g_passwordManager.CheckMenuLock(WINDOW_ADDON_BROWSER))
+    return -1;
+
+  AddonPtr addon;
+  if (!CServiceBroker::GetAddonMgr().GetAddon(addonid, addon, ADDON_UNKNOWN, false) || addon == nullptr)
```

Comment:
You can drop the `addon == nullptr` part, non-null addon is guaranteed if GetAddon() returns true